tracee-ebpf/tracee/filters.go

```diff
@@ -13,6 +13,36 @@ import (
 	bpf "github.com/aquasecurity/libbpfgo"
 )
 
+const (
+	filterNotEqual uint32 = iota
+	filterEqual
+)
+
+const (
+	filterIn  uint8 = 1
+	filterOut uint8 = 2
+)
+
+const (
+	uidLess uint32 = iota
+	uidGreater
+	pidLess
+	pidGreater
```

Comment:
I see that uidGreater, pidGreater, mntNsGreater, and pidNsGreater constants are defined but not used in code. Is it expected?

Reply:
It is used implicitly in line 170 (previously 140) to initialize the bpf map. It is always less+1 of theatching filter